modules/http-server/pom.xml

```diff
@@ -53,6 +53,11 @@
     <packaging>bundle</packaging>
     <version>2.3.36-MULE-026-SNAPSHOT</version>
     <name>grizzly-http-server</name>
+
+    <properties>
+        <mockitoVersion>4.11.0</mockitoVersion>
+    </properties>
```

Comment:
Do these share any common parent where we could put this instead of repeating it in every module?